src/bridgeState.js

```diff
@@ -225,10 +215,12 @@ module.exports = class BridgeState {
     ];
 
     const nodeState = (await this.db.getNodeState()) || {};
+    logVerbose(`Restored node state`, nodeState);
     this.periodProposal = nodeState.periodProposal || null;
     this.stalePeriodProposal = nodeState.stalePeriodProposal || null;
     this.lastSeenRootChainBlock =
       nodeState.rootChainBlockAtProposal || this.genesisBlockHeight;
+    this.blockHeight = nodeState.blockHeight;
```

Comment:
`nodeState.blockHeight` is the plasma height at the time of period proposal creation. "period proposal creation" is every 32nd block, we are replaying from this point on every restart